packages/eslint-plugin-millionjs/README.md

```diff
@@ -0,0 +1,105 @@
+# eslint-plugin-millionjs-drex
```

Comment:
Can you change this to eslint-plugin-million. I can try publishing it on my side. Make sure to change the rest of the instances as well.

Reply:
No problem
I just put it there so everyonbe can install it from the npm registry and test
i'd do just that